task/backend/query_logreader.go

```diff
@@ -114,7 +114,7 @@ from(bucketID: "000000000000000a")
 	|> filter(fn: (r) => r._measurement == "records" and r.taskID == %q)
 	|> drop(columns: ["_start", "_stop"])
 	|> group(columns: ["_measurement", "taskID", "scheduledFor", "status", "runID"])
-	|> influxFieldsAsCols()
+	|> pivot(rowKey:["_time"], columnKey: ["_field"], valueColumn: "_value")
```

Comment:
Why not `import "influxdata/influxdb/v1"` and use `v1.fieldsAsCols()` here instead of `pivot()`?